website/docs/reference/resource-properties/unit-tests.md

```diff
@@ -0,0 +1,203 @@
+---
+title: "About unit tests property"
```

Comment:
![Screenshot 2024-02-14 at 11 01 24 AM](https://github.com/dbt-labs/docs.getdbt.com/assets/53586774/22c644a2-7f29-44b5-adc0-22b5b3423a91)

I can see this on version 1.7 - is that expected?